src/App.jsx

```diff
@@ -1,104 +1,27 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
-
-export const App = () => (
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+const combinePosts = postsFromServer.map(post => {
+  const user = usersFromServer.find(person => person.id === post.userId);
+  const postComments = commentsFromServer.filter(
+    coments => coments.postId === post.id,
+  );
```

Comment:
Let's improve the code a bit here and split that code into your separate functions.

Example:
```
const getUserById = (userId) => {}
```
```
const getCommentsByPostId = (postId) => {}
```